### PR TITLE
Allow custom snippets directory via JUnit 5

### PR DESCRIPTION
**Pull request by nt-jj**
With these changes it is possible to specify a custom directory where the snippets are written to if you are using JUnit 5.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@nt-jj Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-restdocs&pullRequestId=633)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-restdocs&pullRequestId=633) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@nt-jj Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-restdocs&pullRequestId=633)!

File: docs/src/docs/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -397,6 +397,18 @@ based on your project's build tool:
 
 |===
 
+The default can be overridden by providing an output directory when registering the
```
Comment (wilkinsona) [inline docs/src/docs/asciidoc/getting-started.adoc]: This requires JUnit 5.1 or later (which isn't unreasonable). Could you preface this with something like "If you are using JUnit 5.1 or later,"?

Comment (nt-jj) [reply docs/src/docs/asciidoc/getting-started.adoc]: Done.

**Comment by wilkinsona** [comment]
Thanks very much for the PR. The proposed changes are now in master.

**Pull request closed without merging**